fetch_wp_templates.py

```diff
@@ -0,0 +1,81 @@
+"""
+Generate a mapping between WikiProjects and their templates using English
+Wikipedia's API.
+
+Usage:
+    taxon_statistics (-h | help)
```

Comment:
I believe `taxon_statistics` should be `fetch_wp_templates` in the script call signature